src/components/GameDataModal/GameDataModalInfo/GameDataModalInfo.jsx

```diff
@@ -29,8 +29,15 @@ export default function GameDataModalInfo({ gameData, gameAssets }) {
     return (
         gameData && (
             <div className={styles['game-modal-data']}>
+                {/* prettier-ignore */}
+                <button className={styles['close-modal-btn']} onClick={closeModal}>
+                    <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="none">
```

Comment:
maybe can move to assets and import the svg

Reply:
SVG tag for darkmode purposes